cumulusci/cumulusci.yml

```diff
@@ -1,7 +1,14 @@
 cumulusci:
     keychain: cumulusci.core.keychain.EncryptedFileProjectKeychain
-
+    
 tasks:
+    activate_process_builder:
```

Comment:
Sorry for _not_ getting this suggestion in earlier, but I'm going to propose we standardize on the term "flow" rather than "process builder". This is for two reasons:

1. The underlying metadata is called `FlowDefinition` for the entities that are called Process and Flow in the UI.
2. It seems to be more widely accepted that way.
3. It pluralizes more cleanly.